gui/MetadataView.cxx

```diff
@@ -281,7 +281,10 @@ void MetadataView::updateMetadata(kv::metadata_vector const& mdVec)
 {
   QTE_D();
 
-  for (auto const k : d->itemIds())
+  auto traits = kv::metadata_traits{};
+
+  const auto keys = d->itemIds();
```

Comment:
Nit: s.b. `auto const`, as it is elsewhere.

Reply:
fixed